.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,191 @@
+name: PHPUnit
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+
+    strategy:
+      # do not stop on another job's failure
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
+        databases: ['sqlite']
+        server-versions: ['master']
+  
+    name: php${{ matrix.php-versions }}/${{ matrix.databases }} on ${{ matrix.server-versions }}
+
+    steps:
+      - name: Checkout server
+        uses: actions/checkout@v2
+        with:
+          repository: nextcloud/server
+          ref: test/debug-install
```

Comment:
```suggestion
          ref: ${{ matrix.server-versions }}
```

Reply:
We could even use [`github.ref`](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#github-context)

Reply:
I'd say we keep it like this for now, as a lot of apps do not have the same stable branches like the server does. We can add a comment about that, so we can change it to base_ref for shipped apps.